keycloak-extensions/src/main/resources/META-INF/jboss-deployment-structure.xml

```diff
@@ -0,0 +1,7 @@
+<jboss-deployment-structure>
+    <deployment>
+        <dependencies>
+            <module name="org.keycloak.keycloak-services" />
+        </dependencies>
+    </deployment>
+</jboss-deployment-structure>
```

Comment:
Do you know if this file is actually required?  The example I saw in the `keycloak-extension-playground` project didn't have this, so I wonder if its superfluous.

Reply:
Yeah, it was in the example I originally worked from, so maybe it isn't actually needed. I hadn't tried it without it though.